topics/cluster-spec.md

```diff
@@ -550,6 +565,11 @@ Slots migration is explained in similar terms but with different wording
 (for the sake of redundancy in the documentation) in the `CLUSTER SETSLOT`
 command documentation.
 
+Starting from Valkey 8.0, when the primary in either the source or target shard fails during live reconfiguration,
+the primary in the other shard will automatically attempt to update its migrating/importing state to correctly pair
+with the newly elected primary. If this update is successful, the ASK redirection will continue functioning without
+requiring operator intervention.
```

Comment:
Good.

But to complete the slot migration, the cluster does need manual intervention though, at least if valkey-cli is used, right? I mean if a failover happens while `valkey-cli cluster rehard` or `valkey-cli cluster rebalance` is running, valkey-cli aborts, right? Shouldn't we explain what an admin needs to do to complete the slot migration? `valkey-cli cluster --fix` can handle it?

Should we also mention that replicas are now able to return ASK redirects? I added it already to the READONLY command docs but maybe it's good to mention it briefly here in the ASK section in the cluster spec too?

This is what we have for the READONLY command regarding replicas sending redirects:

       3. Since Valkey 8.0, if a slot migration is ongoing.  In this case  the
          replica  can  return  an ASK redirect or a TRYAGAIN error reply.  In
          earlier versions, replicas are not aware of ongoing slot migrations.

Reply:
Added text looks great. Thanks.